_includes/row-title.html

```diff
@@ -0,0 +1,15 @@
+{%- assign website = include.website -%}
+{%- assign img_path = include.section | append: '/' | append: website.img -%}
+<div class="title">
+  <noscript>
+    <img src="/img/{{ img_path }}" class="icon" alt="{{ website.name }}">
```

Comment:
nit pick: might make sense to put "/img/" into the img_path variable since its repeated in the body here and on L7

Reply:
because on L7 we still have to write out /img/ for the placeholder image it doesnt make that much sense to overly shorten things. It can be done and wouldnt be wrong. I would actually love to play with using the src-set tag to define "/img/" so it gets used for both on Line 7. I just havent messed with it enough to confirm it works as expected

Reply:
I can see someone updating L5 and forgetting L7 or vice versa. The placeholder image path in L7 does also start with "/img/", but doesn't use the variable and is a hardcoded value.

Its definitely a nit pick, but figured I would throw it out there.